docs/pages/application-access/guides/aws-console.mdx

```diff
@@ -287,6 +289,91 @@ To log out of the aws application and remove credentials:
 $ tsh app logout awsconsole-test
 ```
 
+## Step 9. Using other AWS applications
```

Comment:
In our style guide we recommend using `Step 1/N.` so that the reader knows how far along they are.